src/ghdcbot/config/models.py

```
@@ -216,6 +216,24 @@ class SnapshotConfig(BaseModel):
+    enabled: bool = False
+    secret: str = ""
+
+    @field_validator("secret")
+    @classmethod
+    def validate_secret(cls, value: str) -> str:
+        if value and len(value) < 16:
+            raise ValueError("webhooks.secret must be at least 16 characters")
+        return value
+
+    @model_validator(mode="after")
+    def validate_enabled_secret(self) -> "WebhookConfig":
```

Ruff (0.15.12)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)